lib/plugins/create/templates/azure-nodejs/package.json

```diff
@@ -12,6 +12,6 @@
   ],
   "dependencies": {},
   "devDependencies": {
-    "serverless-azure-functions": ">=0.7.0"
+    "serverless-azure-functions": "^1.0.0-26"
```

Comment:
I believe it should be `^1.0.0` (and we should merge it after v1 is released)

Reply:
What about '*'?

Reply:
sounds good